### DISCUSSION TITLE
React vs Vue?

### DISCUSSION THREAD

**Discussion by Dnkfff** [HowProgrammingWorks/LiveQA#34] [Q&A] [answered]
Как Вы считаете, что лучше, **React.js** или **Vue.js**. Пишу на реакте уже _пол_ года с лишним, пришло предложение делать на вуе, но я отказался, потому что имею больше опыта с первым, и так же пока, к сожалению, нет желания лапать новые фреймворки. Как вы считаете, кто в этом поединке победитель, Реакт жс от Цукерберга или Вуе жс?

**Comment by tshemsedinov** [comment, marked as answer]
И то и то - оверинженеринг и технологический тупик. Я не специалист во фронтенде, но мне глубоко отвратительно все, что там наваяли во всех этих ужасных фреймворках для решения банальных задач. Я верю в Web-components, Shadow DOM, PWA и Browser API. А все эти ваши ангулары с реактами и вью - сверните в трубочку...

**Reply by Dnkfff** [reply]
И закурить?

**Comment by Anatolii-Ovcharuk** [comment]
Многие останавливаются на React.js. В связке с Redux - неплох для глобальных задач... Но некоторые компании работают и на Vue.js В общем внимание на React.js., а Vue.js для своего познания чтоб понимать как с ним работать...
